x-pack/plugins/maps/public/shared/layers/layer.js

```diff
@@ -17,8 +17,8 @@ export class AbstractLayer {
     this._descriptor = AbstractLayer.createDescriptor(layerDescriptor);
     this._source = source;
     this._style = style;
-    if (this._descriptor.dataRequests) {
-      this._dataRequests = this._descriptor.dataRequests.map(dataRequest => new DataRequest(dataRequest));
+    if (this._descriptor.__dataRequests) {
+      this._dataRequests = this._descriptor.__dataRequests.map(dataRequest => new DataRequest(dataRequest));
```

Comment:
This seems error prone to have `this._dataRequests` with a single underscore and  `this._descriptor.__dataRequests` with a double underscore. Any ideas on how to avoid?

Reply:
yeah .... it looks funny.

We could just revert to single underscore, and ditch the double underscore. 

On the other hand, it's a pretty common convention to prepend `_` to variables of private state or internal state. But I feel like store-state is something different altogether (just gut feeling..)

Maybe we could just leave as-is it for now (?)

Reply:
yep, leaving as is for now sounds ok.